.github/workflows/build_pypi_package.yaml

```diff
@@ -25,7 +24,6 @@ jobs:
     strategy:
       matrix:
         directory:
-          - "conda-store"
           - "conda-store-server"
     defaults:
```

Comment:
If we are removing `conda-store`, it would be best to remove the `matrix` strategy altogether and refactor the workflow